### PR TITLE
Add Ports section to readme

### PR DESCRIPTION
**Pull request by edouard-lopez**
see [sindresorhus/pure#183](https://github.com/sindresorhus/pure/issues/183).

### CODE REVIEW

**Comment by mafredri** [comment]
I don't know which is better, to list all the ports in the readme and remove them from the Wiki or just link to the Wiki from the readme?

As it stands there is duplicate info and two lists to maintain.

File: readme.md [hunk 1]
```diff
@@ -161,6 +161,17 @@ antibody bundle mafredri/zsh-async
 antibody bundle sindresorhus/pure
 ```
 
+## Shell Ports
```
Comment (mafredri) [inline readme.md]: `Shell` here is superfluous (and could be understood as pure is not for the shell), pure is a shell theme and thus these are just *ports*.

File: readme.md [hunk 2]
```diff
@@ -161,6 +161,17 @@ antibody bundle mafredri/zsh-async
...
+
+* **Bash**
+	* [sapegin/dotfiles](https://github.com/sapegin/dotfiles)’s [prompt](https://github.com/sapegin/dotfiles/blob/master/includes/bash_prompt.bash) and [color theme](https://github.com/sapegin/dotfiles/tree/master/color) for `Terminal.app` ;
```
Comment (mafredri) [inline readme.md]: Extra `;` at the end.

File: readme.md [hunk 3]
```diff
@@ -161,6 +161,17 @@ antibody bundle mafredri/zsh-async
...
+* **Fish**
+	* [brandonweiss/pure.fish](https://github.com/brandonweiss/pure.fish): a Pure-inspired prompt for Fish, not intended to have feature parity.
+	* [rafaelrinaldi/pure](https://github.com/rafaelrinaldi/pure),  support for bare Fish and various framework ([Oh-My-Fish](https://github.com//oh-my-fish/oh-my-fish), [Fisherman](https://github.com//fisherman/fisherman) and [Wahoo](https://github.com//bucaran/wahoo).
```
Comment (mafredri) [inline readme.md]: Missing parenthesis at the end.

**Comment by edouard-lopez** [comment]
I think it better to remove the wiki, just keep a minimal list and let each repo provide screenshots and descriptions.

**Comment by sindresorhus** [comment]
Place it below the FAQ section

**Comment by mafredri** [comment]
Looks good to me :+1:!

**Comment by zmwangx** [comment]
Well, should have been squashed... (I subscribe to pure's commits feed and these three commits in a row including two apparent fixups were pretty hard to make sense of until I came to this PR and saw the full picture.) But since it already landed on master...

**Comment by mafredri** [comment]
Sorry about that @zmwangx. I noticed it myself after I had hit the merge button, but it was too late :anguished:.

**Comment by zmwangx** [comment]
No problem! I know the big merge button is inviting :wink:

**Comment by edouard-lopez** [comment]
I make them through the web interface as they where minor/doc edit. 

For future PR, is there a way to squash them through web UI?

**Comment by sindresorhus** [comment]
@edouard-lopez No, but you could use http://rebaseandsqua.sh/

**Comment by zmwangx** [comment]
There's an unofficial feature request for this: https://github.com/isaacs/github/issues/406.

**Pull request merged**